### DISCUSSION TITLE
请问是否考虑把悬浮球改回老款？

### DISCUSSION THREAD

**Discussion by AmenLi** [immersive-translate/immersive-translate#1741] [General]
老款的悬浮球能附在页面顶部，挺好用的
现阶段只能左右两边，放左边挡字，放右边挡某些网页的右侧功能键及网页滑动条
能不能提供上下吸附的功能选项呢？
我知道有快捷键功能
但是有时候懒人躺，并不像伸手按快捷键。

**Comment by KnIfER** [comment]
倒反天罡哈哈。键盘才是最好的。我关闭悬浮球，于是除了背景js外，对原网站几乎没有任何侵入。

快捷键可以设置成以及一键触发。这样可以让他绑定到鼠标上或者离他近的按键。你可以用油猴脚本自动触发。

我还魔改成。双击右键翻译段落。划词划句打开翻译面板。

沉浸翻译？抹（面）布（板）翻译！哪里不会抹哪里……
